src/dams_mcda/server.R

```diff
@@ -1119,22 +1357,47 @@ server <- function(input, output, session) {
 			Dam1ScoreTable
 		})
 
+		# WSM Download button
+		output[[paste0("DownloadDam", damId, "ScoreTable")]] <- downloadHandler(
+			filename = function() {
+				# format date & time in filename, format( year, month, day, hour, minute, second, UTC offset )
+				format(Sys.time(), "WestEnfield_mcda_results_%Y-%m-%d_%H-%M-%S_%z.csv")
+			},
+			content = function(file) {
+				ScoreTablePlusSum <- Dam1ScoreTable
+				ScoreTablePlusSum$Total = IndScoreSum[damId,]
+				write.csv( ScoreTablePlusSum, file, row.names = TRUE, quote=TRUE)
+			}
+		)
+
 		# (d) has three graphs for each dam
-		# d1 (100% score for each alternative)
-		output[[paste0("WSMPlot", damId, "c")]] <- renderPlot2DScaled100(
-		  t(ResultsMatrix[,,damId]),
-		  "D 3", # title
-		  alternative_names, # x_labels
-		  criteria_names, # x_labels
-		  "Alternative", # x axis label
-		  "Score", # y axis label
-		  "Criteria", # legend label
-		  colors, # colors
-		  NULL # x value limit
+		# d1
+		plotA <- renderPlot2D(

```

Comment:
I think this is the one that got swapped incorrectly. So, this would just need to be changed: 
 
alternative_names, (x_labels, currently as "criteria_names in line 1378)
criteria_names ( y_labels, currently as alternative_names in line 1379)			
"Alternative"  ( x axis label, currently "criteria" in line 1380)				
"Score" (y axis label	)			
"Criteria" (legend label)		
colors

That should work.